File: test_cartpanda_api.js
const token = "eu7RWsK5FioYdoPOmn0eRG2z8svuQFWKOkewhbzF8GjeYcKoexGsAkG6uiUZ";
const url = "https://accounts.cartpanda.com/api/v1/products";

async function fetchProducts() {
    try {
        const response = await fetch(url, {
            headers: {
                "Authorization": `Bearer ${token}`,
                "Accept": "application/json"
            }
        });
        const text = await response.text();
        console.log("Status:", response.status);
        console.log(text.substring(0, 500));
    } catch (e) {
        console.error(e);
    }
}

fetchProducts();

File: test_cartpanda_api.py
import requests
import json

url = "https://luminapet.mycartpanda.com/api/v1/products"
headers = {
    "Authorization": "Bearer eu7RWsK5FioYdoPOmn0eRG2z8svuQFWKOkewhbzF8GjeYcKoexGsAkG6uiUZ",
    "Accept": "application/json"
}

response = requests.get(url, headers=headers)
print("Status Code:", response.status_code)
try:
    print(json.dumps(response.json(), indent=2))
except Exception as e:
    print("Response text:", response.text)
